dipy/align/cpd.py

```diff
@@ -196,7 +201,6 @@ def __init__(
         max_iterations=None,
         tolerance=None,
         w=None,
-        *args,
```

Comment:
Why do you remove *args ?

Reply:
let me fix this, i think it happened while i was merging conflicts for rebasing.